### PR TITLE
V3.2.2

### PR DESCRIPTION
**Pull request by kmuehlbauer**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

Using a copy of https://github.com/SrNetoChan/qgis-feedstock/tree/update_to_3.0.3

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
@SrNetoChan @ocefpaf Trying latest qgis 3.2.2.

**Comment by kmuehlbauer** [comment]
windows patches need adaption

**Comment by kmuehlbauer** [comment]
@ocefpaf Can you restart Circle, if you find the time? I tried without luck, obviously missing some rights.

**Comment by ocefpaf** [comment]
> @ocefpaf Can you restart Circle, if you find the time? I tried without luck, obviously missing some rights.

Done! Circle is a little bit annoying when it comes to re-start. We need to send an empty commit, closing and opening won't work. (Or you need to have the rights to do so, which I think you would if you add yourself as a maintainer here.)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -136,3 +135,4 @@ extra:
   recipe-maintainers:
     - ceholden
     - ocefpaf
+    - kmuehlbauer
```
Comment (ocefpaf) [inline recipe/meta.yaml]: :+1:

**Comment by kmuehlbauer** [comment]
@conda-forge-admin, please rerender

**Comment by kmuehlbauer** [comment]
@ocefpaf This is a tough one. Only `fontconfig` is pulled in from `defaults`. Do we have a means to track down which dependency is pulling this in?

But the problem is still the same. I tracked it down to an inconsistency with `bison` and `m4` but did not get further. Do you expect .la problem still an issue here?

**Comment by kmuehlbauer** [comment]
Hm, in the logs, `/opt/rh/devtoolset-2/root/usr/bin/gcc` (4.8.2) is used, instead of the conda provided 7.2.0.

**Comment by ocefpaf** [comment]
> @ocefpaf This is a tough one. Only `fontconfig` is pulled in from `defaults`. Do we have a means to track down which dependency is pulling this in?

I'm experimenting with builds without `defaults`, let me re-render this with my hacked conda-smithy for that, hold on...

> Hm, in the logs, `/opt/rh/devtoolset-2/root/usr/bin/gcc` (4.8.2) is used, instead of the conda provided 7.2.0.

Yep. We are slowly migrating to the new compilers and that is done in topological order. `qgis` is probably far down the stack so it will take a while to be rebuilt with `gcc7`.

**Comment by kmuehlbauer** [comment]
finally sorted out that M4 issue: https://github.com/conda-forge/bison-feedstock/issues/7

But is stuck with the probable gcc-bug, mentioned by @ceholden here: https://github.com/conda-forge/qgis-feedstock/issues/17#issuecomment-373222272

Seems for linux it's either updating compilers or updating qt...

**Comment by ocefpaf** [comment]
> Seems for linux it's either updating compilers or updating qt...

You can try the new compilers by "hacking" the `conda-forge.yml`.
Add a line with `compiler_stack: comp7` and re-render.

**Comment by kmuehlbauer** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
@kmuehlbauer a lot of the packages will come from `defaults` in that case b/c conda-forge just started migrating to the new compilers. So I'm not confident this will work... Still it is worth trying :smile:

**Comment by kmuehlbauer** [comment]
@ocefpaf The new docker-images stumble over the `yum install` stuff. Giving up for today.

**Comment by ocefpaf** [comment]
> @ocefpaf The new docker-images stumble over the `yum install` stuff. Giving up for today.


I'll try to take a look soon but it is a long weekend here and I'm still recovering from fixing `gdal` :smile:

**Comment by kmuehlbauer** [comment]
@ocefpaf No hurry, I'll not have time before next week.

**Comment by kmuehlbauer** [comment]
@conda-forge-admin, please rerender

**Comment by ceholden** [comment]
Closing this now that 3.4 is up (per comments in other PR)

**Pull request closed without merging**